benchmarks/pipe.py

```diff
@@ -5,17 +5,17 @@
 import math
 import os
 import time
-import warnings
 
 from benchmark_dataset import BenchmarkLMDataset, collate_sentences_lm
+import datasets
+import models
+from models import TransformerLMSequntial
```

Comment:
it is strange to import some class one way and others the other way. perhaps just import models and use models.<foo> would be better?

Reply:
Done.